drake/systems/plants/@Manipulator/parameterEstimation.m

```diff
@@ -34,17 +34,26 @@
 % @option model determines which model to use for estimation
 % 'dynamic'     = use dynamic model - requires qdd
 % 'energetic'   = use energetic model - doesn't require qdd
+% 'simerr'      = minimize simulation error
 % 
 % @option method determines which method to use for estimation
 % 'nonlinprog'    = nonlinear least squares (LS) to solve problem
 % 'linprog'       = linear LS on lumped params then nonlinear LS to recover
```

Comment:
Wait, did you mean I should change the name "linprog" to something that mentions nonlinear least squares as well?  I'm not sure what you were asking for here.